packages/react-dev-utils/package.json

```diff
@@ -28,6 +29,7 @@
     "html-entities": "1.2.0",
     "opn": "4.0.2",
     "sockjs-client": "1.0.3",
-    "strip-ansi": "3.0.1"
+    "strip-ansi": "3.0.1",
+    "webpack": "^1.13.3"
```

Comment:
There isn't an explicit webpack dependency to support both 1.x and 2.x.

See https://github.com/facebookincubator/create-react-app/pull/963 and https://github.com/facebookincubator/create-react-app/pull/950#issuecomment-258514681 for more info.